src/App.jsx

```diff
@@ -1,10 +1,55 @@
-import React from 'react';
-import './App.css';
+import { React, useState } from 'react';
+import axios from 'axios';
+
+const geocodingURI = 'http://api.openweathermap.org/geo/1.0/direct?q=';
+const weatherApiKey = process.env.REACT_APP_WEATHER_API_KEY;
 
 function App() {
+  const [geoData, setGeoData] = useState({});
+
+  const retrieveGeoData = async (city = '') => {
+    try {
+      const response = await axios.get(
+        `${geocodingURI}${city}&appid=${weatherApiKey}`
+      );
+
+      const { data } = response;
+      const [location] = data;
+      const coordinates = {
+        longitude: location.lon,
+        latitude: location.lat,
+      };
+
+      setGeoData(coordinates);
+      // Only console logging geoData so eslint doesn't throw error
+      // remove later when passing geoData as prop
+      console.log('The coordinates are: ');
+      console.log(geoData);
+      return undefined;
```

Comment:
@awong-pdx Can you remove this when integrating your portion of the code? Thank you!